packages/loader/driver-definitions/src/storage.ts

```diff
@@ -221,10 +221,31 @@ export interface IDocumentDeltaConnection extends IEventProvider<IDocumentDeltaC
     removeAllListeners(): void;
 }
 
+export enum ServiceCachingPolicy {
+    /**
+     * Driver does not implement any prefetching or caching policy.
+     */
+    None,
+
+    /**
+     * Driver implements prefetching policy, i.e. it prefetches resources from the latest snapshot.
+     */
+    Prefetch,
+}
+
+export interface IDocumentServicePolicies {
+    readonly caching: ServiceCachingPolicy;
+}
+
 export interface IDocumentService {
 
     resolvedUrl: IResolvedUrl;
 
+    /**
```

Comment:
Why does the policies have to be a part of the driver api?

it seems like it should just be on the constructor of this specific driver since different drivers probably have different policies.

Reply:
@skylerjokiel , can you please clarify question? I'm not sure what you mean.
This is driver interface, we need to put it here such that loader has access to it.
OdspDocumentService  (particular instance) puts actual values in here (in constructor(), implicitly)

Reply:
As I said above... I missed the fact that prefetch happens by wrapping the driver in a PrefetchDocumentStorageService in the Container. I get why this doesn't make sense :)